golem-common/src/model/public_oplog.rs

```diff
@@ -328,6 +330,396 @@ pub enum PublicOplogEntry {
+                    || Self::string_match("exported-function-completed", &[], query_path, query)
+                    || Self::string_match("exported-function", &[], query_path, query)
+                    || Self::match_value(&params.response, &[], query_path, query)
+                // TODO: should we store function name and idempotency key in ExportedFunctionCompleted?
+            }
+            PublicOplogEntry::Suspend(_params) => {
+                Self::string_match("suspend", &[], query_path, query)
+            }
+            PublicOplogEntry::Error(params) => {
+                Self::string_match("error", &[], query_path, query)
+                    || Self::string_match(&params.error, &[], query_path, query)
+            }
+            PublicOplogEntry::NoOp(_params) => Self::string_match("noop", &[], query_path, query),
+            PublicOplogEntry::Jump(_params) => Self::string_match("jump", &[], query_path, query),
+            PublicOplogEntry::Interrupted(_params) => {
+                Self::string_match("interrupted", &[], query_path, query)
+            }
+            PublicOplogEntry::Exited(_params) => {
+                Self::string_match("exited", &[], query_path, query)
+            }
+            PublicOplogEntry::ChangeRetryPolicy(_params) => {
+                Self::string_match("changeretrypolicy", &[], query_path, query)
+                    || Self::string_match("change-retry-policy", &[], query_path, query)
+            }
+            PublicOplogEntry::BeginAtomicRegion(_params) => {
+                Self::string_match("beginatomicregion", &[], query_path, query)
+                    || Self::string_match("begin-atomic-region", &[], query_path, query)
+            }
+            PublicOplogEntry::EndAtomicRegion(_params) => {
+                Self::string_match("endatomicregion", &[], query_path, query)
+                    || Self::string_match("end-atomic-region", &[], query_path, query)
+            }
+            PublicOplogEntry::BeginRemoteWrite(_params) => {
+                Self::string_match("beginremotewrite", &[], query_path, query)
+                    || Self::string_match("begin-remote-write", &[], query_path, query)
+            }
+            PublicOplogEntry::EndRemoteWrite(_params) => {
+                Self::string_match("endremotewrite", &[], query_path, query)
+                    || Self::string_match("end-remote-write", &[], query_path, query)
+            }
+            PublicOplogEntry::PendingWorkerInvocation(params) => {
+                Self::string_match("pendingworkerinvocation", &[], query_path, query)
+                    || Self::string_match("pending-worker-invocation", &[], query_path, query)
+                    || match &params.invocation {
+                        PublicWorkerInvocation::ExportedFunction(params) => {
+                            Self::string_match(&params.full_function_name, &[], query_path, query)
```

Comment:
Minor: is `full_function_name` basically `fully_qualified_name`? May be the latter is more common to use. What do you think?

Reply:
Are you suggesting to change the field name?

Reply:
Yes I was